iroh-store/src/store.rs

```diff
@@ -9,7 +9,6 @@ use iroh_metrics::{
     inc, observe, record,
     store::{StoreHistograms, StoreMetrics},
 };
-use iroh_rpc_client::Client as RpcClient;
```

Comment:
you can probably remove the dep as well?

Reply:
no, it is still used by the config for server in `main.rs`.